tests/vmi_networking_test.go

```diff
@@ -791,6 +793,117 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 				}, 120*time.Second).Should(Succeed())
 			})
 		})
+
+		Context("MTU verification", func() {
+			var vmi *v1.VirtualMachineInstance
+			var anotherVmi *v1.VirtualMachineInstance
+
+			getMtu := func(pod *k8sv1.Pod, ifaceName string) int {
+				output, err := tests.ExecuteCommandOnPod(
+					virtClient,
+					pod,
+					"compute",
+					[]string{"cat", fmt.Sprintf("/sys/class/net/%s/mtu", ifaceName)},
+				)
+				log.Log.Infof("%s mtu is %v", ifaceName, output)
```

Comment:
Is this a debugging leftover?

Reply:
It was copied from the bridge mtu test. But indeed pretty redundant. I will remove.